tool/gravity/cli/rpcagent.go

```diff
@@ -345,6 +389,17 @@ func newDeployAgentsRequest(ctx context.Context, env *localenv.LocalEnvironment,
 	}, nil
 }
 
+// findServerByHostname returns the server with the matching hostname. Returns
+// a NotFound error if a server with a matching hostname does not exist.
+func findServerByHostname(clusterState storage.ClusterState, hostname string) (server storage.Server, err error) {
```

Comment:
I'm pretty sure we already have this function in storage.go.

https://github.com/gravitational/gravity/blob/master/lib/storage/storage.go#L696